opm/output/OutputWriter.hpp

```diff
@@ -74,44 +75,28 @@ class OutputWriter {
      * This routine should be called before the first timestep (i.e. when
      * timer.currentStepNum () == 0)
      */
-    virtual void writeInit(const SimulatorTimerInterface &timer, const NNC& nnc = NNC()) = 0;
+    virtual void writeInit( const NNC& nnc = NNC() ) = 0;
```

Comment:
Quick check in simulators shows that the NNC is always passed regardless, although the discussion probably should go on.